src/components/AccountSettings.tsx

```diff
@@ -209,10 +212,6 @@ export const AccountSettings = () => {
                       value={location}
                     />
                   </FormGroup>
-                  <FormGroup>
-                    <FormLabel>Institutional affiliation</FormLabel>
```

Comment:
Has no onChange() and is not mapped to a UserProfile field.  I think this is supposed to be the Current affiliation (UserProfile.company)